src/components/Q&A/AnswerComp.jsx

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import AnswerReport from './AnswerReport.jsx';
+import {API_KEY} from '../../config/config.js';
+const axios = require('axios');
+
+
+class AnswerComp extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      answerList: this.props.answerList,
+      answerArr: [],
+      totalAnsArr: [],
+      nameSeller: false,
+      reported: false,
+      currentAns: {},
+      moreBtnClick: false
+
+    }
+    this.formatDate = this.formatDate.bind(this);
+    this.checkName = this.checkName.bind(this);
+    this.handleClick = this.handleAnsHelp.bind(this);
+    this.sortByHelp = this.sortByHelp.bind(this);
+    this.handleSeeMoreAns = this.handleSeeMoreAns.bind(this);
+    this.handleSeeLessAns = this.handleSeeLessAns.bind(this);
+  }
+
+
+  formatDate(dateStr){
+    var options = { year: 'numeric', month: 'long', day: 'numeric' };
+    return new Date(dateStr).toLocaleDateString([],options);
+  }
+
+  checkName(nameStr) {
+    if (nameStr === 'Seller') {
+      this.state.nameSeller = true;
+    }
+    return nameStr;
+  }
+  sortByHelp() {
+    var aList = this.state.answerList;
+    var key = Object.keys(this.state.answerList);
+    var arr = [];
+    //populate an array of answer objects at specific key
+  for (var i = 0; i < key.length; i++) {
+    arr.push(aList[key[i]]);
+  }
+  //console.log(arr);
+  //if array length is greater than one meaning there is more than one asnwer
+  if (arr.length > 1) {
+  //compare the helpfullness of each answer and sort it greatest first to lowest last
+    for (var i = 0; i < arr.length; i++) {
+      if (arr[i+1]) {
+        if(arr[i].helpfulness < arr[i+1].helpfulness) {
+          var temp = arr[i];
+          arr[i] = arr[i+1];
+          arr[i+1] = temp;
+        }
+      }
+    }
+    if (this.state.moreBtnClick) {
+      this.setState({totalAnsArr: arr});
+      this.setState({answerArr: arr});
+    } else {
+      this.setState({totalAnsArr: arr});
+      arr = arr.slice(0, 2);
+      this.setState({answerArr: arr});
+    }
+
+  }
+
+  this.setState({answerArr: arr});
+  }
+
+  handleAnsHelp(ans, e) {
```

Comment:
quick reminder to auto fix the indentation (opt + shift + F) and rename any temp vars